### PR TITLE
[AdminListBundle] Interface instead class as argument type

### PR DESCRIPTION
**Pull request by cv65kr**
| Q             | A
| ------------- | ---
| Bug fix?      | kind of
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | no

### CODE REVIEW

File: src/Kunstmaan/AdminListBundle/AdminList/Configurator/AbstractDoctrineORMAdminListConfigurator.php [hunk 1]
```diff
@@ -46,10 +45,10 @@
     protected $aclHelper = null;
 
     /**
-     * @param EntityManager $em        The entity manager
+     * @param EntityManagerInterface $em        The entity manager
```
Comment (sandergo90) [inline src/Kunstmaan/AdminListBundle/AdminList/Configurator/AbstractDoctrineORMAdminListConfigurator.php]: Indentation

**Comment by cv65kr** [comment]
@sandergo90 done.

File: src/Kunstmaan/AdminListBundle/AdminList/Configurator/AbstractDoctrineORMAdminListConfigurator.php [hunk 2]
```diff
@@ -43,10 +43,12 @@
...
      */
     protected $aclHelper = null;
-
+    
     /**
-     * @param EntityManagerInterface $em        The entity manager
-     * @param AclHelper     $aclHelper The acl helper
+     * AbstractDoctrineORMAdminListConfigurator constructor.
+     *
+     * @param EntityManagerInterface $em
```
Comment (sandergo90) [inline src/Kunstmaan/AdminListBundle/AdminList/Configurator/AbstractDoctrineORMAdminListConfigurator.php]: Parameters should be alined.

**Comment by cv65kr** [comment]
@sandergo90, check now please i focused too much on storm doc generator :)

**Comment by sandergo90** [comment]
@cv65kr perfect!

**Pull request merged**